web/src/search/input/SearchFilterChips.tsx

```diff
@@ -159,7 +160,13 @@ export class SearchFilterChips extends React.PureComponent<Props> {
                 value,
             },
         })
-        submitSearch(this.props.history, toggleSearchFilter(this.props.query, value), 'filter')
+
+        let newQuery = toggleSearchFilter(this.props.query, value)
+        if (this.props.isSourcegraphDotCom && value.match(/\brepogroup:/) && newQuery.match(/\brepogroup:sample\b/)) {
```

Comment:
```suggestion
        if (this.props.isSourcegraphDotCom && /\brepogroup:/.test(value) && /\brepogroup:sample\b/.test(newQuery)) {
```
Use `test()` if you just want a boolean